Modules/IO/MeshGifti/src/itkGiftiMeshIO.cxx

```diff
@@ -258,15 +258,15 @@ GiftiMeshIO::ReadMeshInformation()
           this->m_UpdatePointData = true;
           this->m_PointPixelComponentType = GetComponentTypeFromGifti(m_GiftiImage->darray[ii]->datatype);
           this->m_PointPixelType = GetPixelTypeFromGifti(m_GiftiImage->darray[ii]->datatype);
-          this->m_NumberOfPointPixelComponents = GetNumberOfPixelComponentsFromGifti(m_GiftiImage->darray[ii]->datatype);
+          this->m_NumberOfPointPixelComponents =
```

Comment:
All these style commits at the end should be squashed into one commit with `STYLE: ` prefix in commit message.